### ISSUE TITLE
Accessibility error in plone.app.workflow

### ISSUE THREAD

**Issue by federicadelia** [plone/Products.CMFPlone#644]
In the sharing view, checkboxes have no label. It is acceptable to have even the title attribute (the title of the column).

**Comment by polyester** [comment]
Hi @federicadelia, are you testing against Plone 5 or 4.x?

**Comment by federicadelia** [comment]
Hi @polyester, in Plone 4.3.4.1

**Comment by mauritsvanrees** [comment]
Closed due to long inactivity. Please reopen if this is still relevant for Plone 5.2 or 6.
